code/obj/item/organs/lung.dm

```diff
@@ -110,6 +111,10 @@
 			donor.take_toxin_damage(min(ratio * 125,20) * mult/LUNG_COUNT)
 			update.show_tox_indicator = TRUE
 
+		if (Radgas_pp > 0) //any fallout is too much fallout
```

Comment:
mkae cyberlungs protect?

Reply:
I'm not sure they should tbh, radioactive lungs is bad whether or not you have flesh lungs or metal lungs. Plus it gives it a bit of a distinction between fallout and plasma.